core/keyboard_nav/flyout_navigation_policy.ts

```diff
@@ -21,15 +21,17 @@ export class FlyoutNavigationPolicy<T> implements INavigationPolicy<T> {
   constructor(
     private policy: INavigationPolicy<T>,
     private flyout: IFlyout,
-  ) {}
+  ) {
+    console.log('make flyout nav policy');
```

Comment:
Probably want to remove this debug line.

Ditto for the others in this file.

Reply:
Done